pkg/kuberesource/kuberesource.go

```diff
@@ -14,12 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package kube
+package kuberesource
 
 import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-var PodsGroupResource = schema.GroupResource{Group: "", Resource: "pods"}
-var JobsGroupResource = schema.GroupResource{Group: "batch", Resource: "jobs"}
-var NamespacesGroupResource = schema.GroupResource{Group: "", Resource: "namespaces"}
+var Pod = schema.GroupResource{Group: "", Resource: "pods"}
```

Comment:
I think these should all be plural. WDYT @skriss ?

Reply:
I actually started it that way, then got skeptical of myself.

I think it makes sense to be plural and match the actual `Resource` field.

Reply:
plural sounds good to me.